cli/printer/text.go

```diff
@@ -171,6 +171,32 @@ func (t *TextPrinter) PrintLoadBalancers(loadBalancers ...*models.LoadBalancer)
 	return nil
 }
 
+func (t *TextPrinter) PrintLoadBalancerHealthCheck(loadBalancer *models.LoadBalancer) error {
+	getEnvironment := func(l *models.LoadBalancer) string {
+		if l.EnvironmentName != "" {
+			return l.EnvironmentName
+		}
+
+		return l.EnvironmentID
+	}
+
+	rows := []string{"LOADBALANCER ID | LOADBALANCER NAME | ENVIRONMENT | TARGET | INTERVAL | TIMEOUT | HEALTHY THRESHOLD | UNHEALTHY THRESHOLD "}
```

Comment:
If anything, I think we'd show just `TARGET`. If we do, it would be nice to take something out to prevent that table from getting too long